src/aggregate.py

```diff
@@ -0,0 +1,87 @@
+import csv
+import datetime
+import os
+
+import constants
+from helpers import data
+from helpers import merging
+
+
+def load_data_dir(dir_name):
+  return data.combine_all_csvs(os.path.join(os.path.dirname(__file__), '../data', dir_name))
+
+
+def write_unknown_facilities(unknown_facilities):
+  with open(os.path.join(os.path.dirname(__file__), '../output/unknown_facilities.csv'), 'w', encoding='utf-8') as f:
+    csv_writer = csv.writer(f)
+
+    csv_writer.writerow(['State', 'Facility name', 'Source using this name'])
+    for row in set(unknown_facilities):
+      csv_writer.writerow(row)
+
+def write_merged_data(merged_data):
+  with open(os.path.join(os.path.dirname(__file__), '../output/merged_data.csv'), 'w', encoding='utf-8') as f:
+    csv_writer = csv.DictWriter(f, fieldnames=constants.OUTPUT_COLUMNS)
+
+    csv_writer.writeheader()
+
+    for row in merged_data:
+      csv_writer.writerow(row)
+
+
+def aggregate():
+  name_mapper = data.Mapper(data.fetch_csv(constants.MAPPINGS_URL))
+
+  datasets = {'covidprisondata.com': load_data_dir('covidprisondata.com'),
+              'UCLA Law Behind Bars': load_data_dir('ucla'),
+              'Recidiviz': data.fetch_csv(constants.RECIDIVIZ_DATA_URL, as_dicts=True)}
+
+  all_unknown_facilities = []
+
+  for dataset_id, dataset in datasets.items():
+    print('Num rows in %s: %s' % (dataset_id, len(dataset)))
+
+    standardized_dataset, unknown_facilities = data.standardize_dataset(dataset, dataset_id, name_mapper)
+
+    datasets[dataset_id] = data.key_dataset(standardized_dataset)
+    all_unknown_facilities += unknown_facilities
+
+  write_unknown_facilities(all_unknown_facilities)
```

Comment:
Can we add more print statements throughout for better visibility, e.g. in case something hangs? Having a statement before and after any I/O so the operator knows where we are at all times would be great.